app/models/team.rb

```diff
@@ -3,14 +3,14 @@ class Team < ActiveRecord::Base
 
   KINDS = %w(sponsored voluntary)
 
-  validates :kind, presence: true
+  #validates :kind, presence: true, if: :selected?    this validation kicks in when admin sets is_selected to true. to avoid this has been commented
```

Comment:
Ok, I don't think the selection flag should be part of this PR. I will remove it during the merge and we can add it in a seperate PR.